xbmc/addons/AddonDatabase.h

```diff
@@ -74,12 +74,18 @@ class CAddonDatabase : public CDatabase
   bool Search(const std::string& search, ADDON::VECADDONS& items);
 
   /*! \brief Disable an addon.
-   Sets a flag that this addon has been disabled.  If disabled, it is usually still available on disk.
-   \param addonID id of the addon to disable
-   \param disable whether to enable or disable.  Defaults to true (disable)
+   Sets a flag that this addon has been disabled.  If disabled, it is usually still available on
```

Comment:
can you define one line for each of the \params / \return / \sa ?

Reply:
Add then also a `*` before, without make clang this not wanted changes.